tests/test_api_v1.py

```diff
@@ -85,6 +85,12 @@ def test_fetch_paginated_requests(mock_chain, app, auth_env, client, db):
     # Start at 10 because each page contains 10 items and we're processing the second page
     for repo_number, request in enumerate(fetched_requests, 10):
         assert request['repo'] == repo_template.format(repo_number)
+    pagination_metadata = response['meta']
+    assert pagination_metadata['next'].endswith('?page=3&per_page=10')
+    assert pagination_metadata['last'].endswith('?page=5&per_page=10')
+    assert pagination_metadata['first'].endswith('?page=1&per_page=10')
+    assert pagination_metadata['prev'].endswith('?page=1&per_page=10')
```

Comment:
Is the order of the query parameters always guaranteed? I'd be worried that it's not depending on the system and it could cause these tests to fail.

Reply:
I don't think the order is always guaranteed but the order shouldn't matter in the tests since they access individual keys and don't check for the order.